### PR TITLE
add patch to deploy cf on Azure.

### PR DESCRIPTION
**Pull request by andyliuliming**

### CODE REVIEW

File: plan-patches/cf-azure/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Patch: cf-azure
+
+Steps to deploy cf with bbl:
+
+1. Follow the normal steps to bbl up with a patch.
+
+    ```bash
+    export BOSH_BOOTLOADER=<YOUR BOSH BOOTLOAER PATH>
```
Comment (ciriarte) [inline plan-patches/cf-azure/README.md]: Do you think we can fix this small typo? BOOTLOAER -> BOOTLOADER

File: plan-patches/cf-azure/ops/rename-network-and-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+- type: replace
+  path: /instance_groups/name=diego-api/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=uaa/networks/name=default/name
+  value: ((network_name))
+
+# - type: replace
```
Comment (ciriarte) [inline plan-patches/cf-azure/ops/rename-network-and-deployment.yml]: Could you please remove these comment lines if they are not required?

File: plan-patches/cf-azure/ops/rename-network-and-deployment.yml [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+      network: ((network_name))
+      domain: bosh
+  - domain: bits.service.cf.internal
+    targets:
+    - query: '*'
+      instance_group: bits
+      deployment: ((deployment_name))
+      network: ((network_name))
+      domain: bosh
+  # - domain: blobstore.service.cf.internal
```
Comment (ciriarte) [inline plan-patches/cf-azure/ops/rename-network-and-deployment.yml]: Did you mean to leave these comments as an example aliasing a blobstore? Or can we remove them?

Comment (andyliuliming) [reply plan-patches/cf-azure/ops/rename-network-and-deployment.yml]: removed the commented out lines. thanks.

File: plan-patches/cf-azure/ops/rename-network-and-deployment.yml (hunk 2 above)
Comment (andyliuliming) [reply plan-patches/cf-azure/ops/rename-network-and-deployment.yml]: changed. thanks.

File: plan-patches/cf-azure/README.md (hunk 1 above)
Comment (andyliuliming) [reply plan-patches/cf-azure/README.md]: changed. thanks.

**Comment by ciriarte** [comment]
Thanks for addressing the feedback so quickly, @andyliuliming!

**Pull request merged**